amulet/deployer.py

```diff
@@ -10,7 +10,10 @@
 from path import path
 from path import tempdir
 
-from .helpers import default_environment, juju, timeout as unit_timesout
+from .helpers import default_environment
```

Comment:
It would be better to do

```
from .helpers import (
    default_environment,
    juju,
    get_charm_directory,
    timeout as unit_timesout,
)
```

This is a style convention